src/docfx/Models/BuildJsonConfig.cs

```diff
@@ -96,5 +58,9 @@ public class BuildJsonConfig
         /// </summary>
         [JsonProperty("noLangKeyword")]
         public bool NoLangKeyword { get; set; }
+
+        [JsonProperty("docsets")]
+        public List<BuildJsonConfigCommon> Docsets { get; set; }
```

Comment:
but you moved some properties to `BuildJsonConfigCommon`, shouldn't the old docfx.json be updated to contain a `docsets` property with a list that has only one element?